cmd-record.c

```diff
@@ -245,46 +286,25 @@ static void setup_child_environ(struct opts *opts, int pfd)
 	if (opts->patt_type != PATT_REGEX)
 		setenv("UFTRACE_PATTERN", get_filter_pattern(opts->patt_type), 1);
 
-	if (opts->lib_path) {
-		snprintf(buf, sizeof(buf), "%s/libmcount/", opts->lib_path);
+	libpath = get_libmcount_path(opts);
+	if (libpath == NULL)
+		pr_err_ns("cannot found libmcount.so\n");
```

Comment:
When does it return NULL?

Reply:
Hum.. it can return NULL at 16f0afb
So, I'll move this error handling code to 16f0afb